org.adempiere.ui/src/org/compiere/apps/AbstractProcessCtl.java

```diff
@@ -21,12 +21,15 @@
 import org.adempiere.util.IProcessUI;
 import org.adempiere.util.ProcessUtil;
 import org.compiere.model.MPInstance;
-import org.compiere.model.MRule;
 import org.compiere.model.MPInstance.PInstanceInfo;
+import org.compiere.model.MRule;
+import org.compiere.model.Query;
+import org.compiere.print.MPrintFormat;
 import org.compiere.print.ReportCtl;
 import org.compiere.process.ProcessInfo;
 import org.compiere.process.ProcessInfoUtil;
 import org.compiere.util.CLogger;
+import org.compiere.util.DB;
 import org.compiere.util.Env;
 import org.compiere.util.Ini;
 import org.compiere.util.Msg;
```

Comment:
Please fix the following warnings:
- The import org.compiere.model.Query is never used
- The import org.compiere.util.DB is never used